lua/kommentary/config.lua

```diff
@@ -81,11 +100,13 @@ Get the full config for the given filetype.
 	Full configuration for filetype
 ]]
 function M.get_config(filetype)
+    local use_fallback = false
     if filetype == 0 then
         filetype = vim.bo.filetype
+        use_fallback = true
```

Comment:
```suggestion
    if filetype == 0 then
        filetype = vim.bo.filetype
```

If I'm reading this correctly, `use_fallback` is not necessary, since config_from_commentstring will already return default when it can't work it out from the `commentstring`.